backend/api/pretix/query.py

```diff
@@ -146,9 +148,11 @@ def get_conference_tickets(
     quotas = pretix.get_quotas(conference)
 
     def sort_func(ticket):
-        # If the item has variations, it means it is the
-        # t-shirt product. We want to show it always at the bottom
-        if len(ticket.variations) > 0:
+        # Make gadgets and association appear at the end
```

Comment:
Maybe we can turn the env variable into a list containing both `["Gadget", "Association"]` so if we add more categories in the future at the bottom we don't have to change the code?

Reply:
it isn't an environment variable but a constant defined here: https://github.com/pythonitalia/pycon/pull/2547/files#diff-5762a1b4022b3f800641216902ef7c088ac8a70c0d37057eff136650db2cb697R1

Reply:
I don't have a preference to be honest.

If I were to change this code I would split the if instead of using `or` to make it more clear